plugins/woocommerce-docs/tests/src/Manifest/ManifestProcessorTest.php

```diff
@@ -37,7 +37,7 @@ function ( $preempt, $args, $url ) use ( $md_file ) {
 			3
 		);
 
-		ManifestProcessor::process_manifest( $manifest, 123 );
+		ManifestProcessor::process_manifest( $manifest, null, 123 );
```

Comment:
We should re-arrange the arguments for `process_manifest` so that the optional ones come last. Passing in `null` on purpose feels like a sub-optimal work-around.

Reply:
I see what you're saying, and I'm ok with it. Something feels weird about the order if it goes from the below to `( $logger_action_id, $manifest, $previous_manifest )` when all other functions have `$logger_action_id` as the last argument.

```
public static function process_manifest( $manifest, $previous_manifest, $logger_action_id ) { ... }
```

Reply:
I'd prefer the slightly odd `$manifest, $logger_action_id, $previous_manifest` order over the `null` arguments. Alternatively we could reorder things in all methods to have the `$logger_action_id` always come first. What do you think?

Reply:
>I'd prefer the slightly odd $manifest, $logger_action_id, $previous_manifest order over the null arguments. 

I went with this